### PR TITLE
Replace CR LF with a html BR tag for the software version comments

### PR DESCRIPTION
**Pull request by vrdriver**
Replace CR LN with a html br tag for the version comments.

| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #3039

*Please update this template with something that matches your PR*

### CODE REVIEW

File: inc/softwareversion.class.php [hunk 1]
```diff
@@ -329,8 +329,8 @@ static function showForSoftware(Software $soft) {
                echo "<td class='right'>".Dropdown::getDropdownName('glpi_operatingsystems',
                                                                    $data['operatingsystems_id']);
                echo "</td>";
-               echo "<td class='numeric'>$nb</td>";
-               echo "<td>".$data['comment']."</td></tr>\n";
+               echo "<td class='numeric'>$nb</td>";
```
Comment (orthagh) [inline inc/softwareversion.class.php]: warning, you added some trailing spaces after the end of this line (tests fail because that)

Comment (vrdriver) [reply inc/softwareversion.class.php]: Apologies. First time I've done this on github.

**Comment by orthagh** [comment]
last thing, @vrdriver, could you post here (and on the original issue) a screeshot of the change ?

**Comment by vrdriver** [comment]
Please note the new formatting of the notes for both of these images. Before and After.
screenshot 2017-10-27 23 13 13
screenshot 2017-10-27 23 13 27

**Comment by orthagh** [comment]
Perfect. LGTM for merging

**Comment by vrdriver** [comment]
Great. Well there's a first for me. A pull request and commit on github to an open source project. cheers. Keep up the good work with GLPI. 👍

**Comment by trasher** [comment]
Please do not close until it's merged ;)

Same for refering tickets :)

**Pull request merged**